volatility3/framework/plugins/windows/strings.py

```diff
@@ -185,11 +214,14 @@ def generate_mapping(
                             for val in range(
                                 mapped_offset, mapped_offset + mapped_size, 0x1000
                             ):
-                                cur_set = reverse_map.get(mapped_offset >> 12, set())
+                                cur_set = reverse_map.get(mapped_offset, set())
                                 cur_set.add(
-                                    (f"Process {process.UniqueProcessId}", offset)
+                                    (
+                                        f"Process {process.UniqueProcessId}",
+                                        mapped_offset,
+                                    )
                                 )
-                                reverse_map[mapped_offset >> 12] = cur_set
+                                reverse_map[offset] = cur_set
```

Comment:
Again, the difference between `mapped_offset` and `offset` is concerning here...  5:S